src/lib.rs

```diff
@@ -163,6 +165,29 @@ impl Order {
         self.account.get(&mut self.nonce, challenge_url).await
     }
 
+    /// Poll the order with exponential backoff until in a final state
```

Comment:
nit: missing a `.` on the end of this headline sentence?

Reply:
Nope! You'll see that libraries which I originated consistently omit the `.` on the first line, on account of this being often more like a sentence fragment then a full sentence.

Reply:
Ah! Okie